content/docs/gke/authentication.md

```diff
@@ -183,6 +183,79 @@ By default, `Roles` and `RoleBindings` apply only to resources in a specific nam
 You can find more information in the 
 [Kubernetes docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole).
 
+### Namespaces
+
+Starting from version 0.6, you can consume Kubeflow in individual namespaces. In order to do this you will need to
```

Comment:
HOw about adding a bit more detail explaining the namespace model.

i.e. explain that namespace "kubeflow" is intended for running kubeflow system components and users are expected to run individual jobs and model deployments in their own namespace.

Reply:
Done.